code/modules/projectiles/guns/manufacturer/clip_lanchester/ballistics.dm

```diff
@@ -130,7 +130,7 @@
 
 /obj/item/gun/ballistic/automatic/pistol/cm357
 	name = "\improper CM-357"
-	desc = "A semi-automatic magnum handgun designed specifically for BARD's megafauna removal unit, as standard handguns had proven useless as backup weapons. Its heft and power have also made it a status symbol among the few CLIP officers able to requisition one. Chambered in .357."
+	desc = "A powerful semi-automatic magnum handgun, designed for use by BARD's megafauna removal unit, as standard handguns had proven ineffective as backup weapons. The heft and power of the weapon have made it a status symbol among the few CLIP officers able to requisition one. Chambered in .357."
```

Comment:
The changes to the first sentence flow a lot worse, in my opinion; too many commas make the sentence feel really long and unwieldy, and "powerful semi-automatic magnum handgun" is kind of a mouthful. here's a version that slims the first sentence down a bit while keeping the important parts:

"A powerful semi-automatic handgun designed for BARD's megafauna removal unit, as standard handguns had proven ineffective as backup weapons. The heft and power of the weapon have made it a status symbol among the few CLIP officers able to requisition one. Chambered in .357."